src/driver/mongodb.rs

```diff
@@ -139,9 +105,8 @@ impl From<Bson> for Value {
       Bson::JavaScriptCode(value) => Value::String(value),
       Bson::JavaScriptCodeWithScope(value, _) => Value::String(value),
       Bson::I32(value) => Value::I64(i64::from(value)),
-      Bson::I64(value) => Value::I64(i64::from(value)),
+      Bson::I64(value) => Value::I64(value),
       Bson::TimeStamp(value) => Value::I64(i64::from(value)),
-      // TODO: Actual transformation of binary type.
       Bson::Binary(_, _) => Value::Null,
```

Comment:
Int would be the closest one, but it involves some dealing with the bytes, and probably knowing what the types are supposed to represent (bit wise)